src/Katip/Scribes/Raven.hs

```diff
@@ -43,3 +45,6 @@ mkRavenScribe sentryService permitItem _ = return $
 
     sentryName :: Katip.Namespace -> T.Text
     sentryName (Katip.Namespace xs) = T.intercalate "." xs
+
+    extras :: (HM.HashMap T.Text Aeson.Value -> HM.HashMap String String)
```

Comment:
I don't think we want to use `show` here. Strings should be verbatim and other JSON objects should use their JSON syntax (`encode`).

Reply:
This still escapes the `String` values, which I don't think is desirable.

Reply:
Doesn't that introduce ambiguity if the value is a string or a number (or just any other JSON type)? Not sure if removing quotes improves much.

Reply:
It does, but I don't think we need to distinguish them, because they are for human consumption and they're tied to keys that are typically of a single JSON type.
It's not just quotes but also escaping which will probably be annoying.

Reply:
> because they are for human consumption 

I don't see why humans can't parse quotes.

> It's not just quotes but also escaping which will probably be annoying.

I think escaping is only visible in GHCI due to string type, but when you render the string (as sent to sentry), it won't be escaped (otherwise JSON would be invalid?). Maybe I'm wrong, but that would have been weird to me.

Reply:
We render things like Warp Request as JSON, it's going to look weird (and invalid JSON) if we just skip the quotes all over.

Reply:
What katip does is convert things to json and then smash their attributes together into a single set of JSON fields. So talking about a warp request as if it's still one entity rather than some spread out bits of information doesn't seem to make much sense.
As far as I understand, the raven-haskell library will take care of serializing the `Map String String` as appropriate for the sentry API so I don't see why we need to be concerned about JSON validity. For example, this isn't valid JSON either: https://pub.dev/documentation/sentry_client/latest/api_data_sentry_packet/SentryPacket/extra.html

Reply:
Because it will look like JSON, but it won't be JSON. I'm only arguing that your requirement for this change makes little sense to me as in why it's so important to avoid quotes. I hope we agree about escaping?

Reply:
Can we deprioritize this as the goal here is to expose information to error reporting and we can handle how it's represented once we're annoyed by it (if even).

Reply:
I don't think we agree about escaping, but that's ok. We can optimize it later. Having to unescape between copying and pasting is probably not so bad that it needs more discussion etc.

Reply:
If escaping will be an issue I'll fix it :)